app/transformers/api/v4/user_transformer.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V4::UserTransformer
+  class << self
+    def to_response(user)
+      { 'user' =>
```

Comment:
Should we move the `user` nesting to the controller? It will keep all of our key nesting (`user`, `errors`) in one place, and allow our transformers to focus on simply serializing models.